src/polymer/polymer-element.ts

```diff
@@ -168,6 +169,11 @@ export class ScannedPolymerElement extends ScannedElement implements
     if (options && options.methods) {
       options.methods.forEach((m) => this.addMethod(m));
     }
+    if (this.jsdoc) {
+      const summaryTag = jsdoc.getTag(this.jsdoc, 'summary');
```

Comment:
`jsdoc.getTag(this.jsdoc, 'summary', 'description');` should give you the description you're trying to access below.